src/components/elements/CommonMenuButton.tsx

```diff
@@ -39,7 +40,7 @@ const CommonMenuButton = ({
   iconButton,
   MenuProps,
   ...buttonProps
-}: Props) => {
+}: Props & ButtonProps) => {
```

Comment:
If we need to accept all the button props, it might be a bit cleaner to accept a prop `ButtonProps` instead of inheriting them all like this. The reason is that IconButton and Menu components might share a lot of the same props, so it's not clear to the caller which element their prop will be applied to. Clearer is: `<CommonMenuButton ButtonProps={{}} MenuProps={{}} />`
With adding this, it also makes the `variant`,`disabled`,`sx` props above redundant, so those could be removed.